STT/Dockerfile.train

```diff
@@ -0,0 +1,233 @@
+# Build KenLM 
+FROM nvcr.io/nvidia/tensorflow:20.06-tf1-py3 AS kenlm-build
+
+ENV DEBIAN_FRONTEND=noninteractive
+
+RUN apt-get update && \
+  apt-get install -y --no-install-recommends \
+  build-essential cmake libboost-system-dev \
+  libboost-thread-dev libboost-program-options-dev \
+  libboost-test-dev libeigen3-dev zlib1g-dev \
+  libbz2-dev liblzma-dev && \
+  rm -rf /var/lib/apt/lists/*
+
+# Build KenLM to generate new scorers
+WORKDIR /code
+COPY STT/kenlm /code/kenlm
```

Comment:
I dont think we want to have `STT` as a submodule and copy here, you should rather keep the old way of git clone in the docker itself

Reply:
Thank's for the heads up! I'll make the required changes.